ci/vmtest/run_selftests.sh

```diff
@@ -81,13 +102,22 @@ echo "ALLOWLIST: ${ALLOWLIST}"
 
 cd ${PROJECT_NAME}/selftests/bpf
 
-if [ $# -eq 0 ]; then
+# populate TEST_NAMES_FROM_BOOT
+read_tests_from_boot_parameters
+# Sort and only keep unique test names from both boot params and arguments
+# test_names will contain a sorted list of uniq tests to be ran.
+readarray -t test_names < \
+    <(printf '%s\0' "${TEST_NAMES_FROM_BOOT[@]}" "$@" | sort -z -u | xargs -0n1)
```

Comment:
Would it make more sense to move this logic into `read_tests_from_boot_parameters`? Having duplicated tests doesn't really make sense to me, to begin with.

Can we use long arguments, please? I think code should be read- and understandable without having two `man` pages handy at all times, to the degree that is possible with shell scripts.

Reply:
> Would it make more sense to move this logic

We can, as long as we make sure to call this last, we should be fine.

> Having duplicated tests doesn't really make sense to me, to begin with.

Agreed, because we have some of the logic that leaves in libbpf/ci (prepare-rootfs/run-qemu) and here, there is a time window when a mix of both passing through a parameter or boot param may happen.
I rather be conservative here and deal with potential dups making it in rather than ending up running the test twice.

> Can we use long arguments, please?

yes, that's doeable for commands that supports long args (e.g sort and xargs in this case). Will update.